sa/sa_test.go

```diff
@@ -1642,45 +1642,113 @@ func TestGetAuthorizationNoRows(t *testing.T) {
 }
 
 func TestGetAuthorizations(t *testing.T) {
+	if os.Getenv("BOULDER_CONFIG_DIR") != "test/config-next" {
```

Comment:
Why do you need the config gating here?

Reply:
Before #3564 was merged calling `NewOrder` like is done in this updated unit test caused an error because one of the `order` table config-next migrations from `sa/_db-next/` wasn't applied.

It should be possible to remove this now that 3564 landed. I'll do that tomorrow :+1:

Reply:
Resolved in ee05336